recipes-bsp/tegra-binaries/tegra-helper-scripts/initrd-flash.sh

```diff
@@ -252,11 +253,12 @@ prepare_for_rcm_boot() {
     if [ $have_odmsign_func -eq 1 ]; then
 	if [ -e "rcmbootcmd.txt" ]; then
 	    sed -e"s,boot.img,initrd-flash.img," rcmbootcmd.txt > rcm-boot.sh
+	    sed -e"s,${DTBFILE},${INITRD_DTBFILE}," rcmbootcmd.txt > rcm-boot.sh
 	    chmod +x rcm-boot.sh
 	elif [ "$CHIPID" = "0x18" ]; then
 	    local binsfx=".encrypt"
 	    local ksfx=".encrypt"
-	    local kdtbfilebase=$(basename kernel_$DTBFILE .dtb)
+	    local kdtbfilebase=$(basename $INITRD_DTBFILE .dtb)
```

Comment:
Here's where we should fall back to `kernel_$DTBFILE` if `$INITRD_DTBFILE` is null.